rllib/models/torch/torch_action_dist.py

```diff
@@ -305,3 +309,95 @@ def sample(self):
     @override(ActionDistribution)
     def required_model_output_shape(action_space, model_config):
         return np.prod(action_space.shape)
+
+
+class TorchMultiActionDistribution(TorchDistributionWrapper):
+    """Action distribution that operates for list of (flattened) actions.
+
+    Args:
+        inputs (Tensor list): A list of tensors from which to compute samples.
+    """
+
+    def __init__(self,
+                 inputs,
+                 model,
+                 *,
+                 child_distributions,
+                 input_lens,
+                 action_space,
+                 action_space_struct=None):
+
+        if not isinstance(inputs, torch.Tensor):
+            inputs = torch.Tensor(inputs)
```

Comment:
How does this work if the inputs are of different shapes? Does torch still combine them into a tensor?

Reply:
`inputs` is currently (and I guess it should stay this way) always a single ("super-flat") tensor. 
Hence we need `input_lens` (to split this tensor by).

Reply:
Got it. I think the documentation should be updated here then (it currently claims inputs is a "Tensor list").

Reply:
Done.

Reply:
It would work in both cases now: 1) simple list of child-distributions + nested action space 2) nested struct of child distributions + equally nested action space.